backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt

```diff
@@ -79,6 +79,15 @@ class DataCorrectnessRoutesTest : ServerTestBase() {
         // by calling the API and requesting data for patient1.id and expecting Ok status
         // and then comparing correctnessId with the ID you received in the test
         // hint: see first part of the test
+
```

Comment:
you can remove the TODO and the instructions part of the text, just leave the description